swagger2.0/oic.sec.doxm.swagger.json

```diff
@@ -252,12 +252,6 @@
           ]
         },
 
-        "owned" :
```

Comment:
It looks like in addition to removing "owned" from the UPDATE part (since it is changing to read only), you need to also remove "n", "id" and "if" which are also read only.

Reply:
"rt" needs to be removed as well.

Reply:
Updated. How did it pass the schema validation before?